source/includes/k8s/deploy-operator.rst

```diff
@@ -50,6 +50,8 @@ Each pod runs three containers:
 - SideCar container that monitors configuration secrets for the tenant and updates them as they change.
   This container also monitors for root credentials and creates an error if it does not find root credentials. 
 
+Starting with v5.0.6, the MinIO Operator supports custom :kube-docs:`init containers <concepts/workloads/pods/init-containers>` for additional pod initialization that may be required for your environment.
```

Comment:
I'm OK with just saying this, *but* I think we will get a follow-on of "how do I do this".

I am *guessing* either via modifying the tenant YAML manually *or* through the Operator Console - *assuming* we already support that UI/UX. If not would it only be via YAML?

Reply:
> I think we will get a follow-on of "how do I do this".

Someone will want more, for sure. From the conversation with Celis, it involves creating a new container... and after that is where I get lost. So far, the only reported use for a special init container is to fix file permissions in a particular type of setup. And this change allows someone to do that if they need to.

Celis also suggested documenting what our MinIO init container does, perhaps info about custom ones could be included if we did that.

I'll open a docs issue for this, although it is likely to sit in the backlog a while.